app/main/views/briefs.py

```diff
@@ -183,6 +220,43 @@ def update_brief_remove_user(brief_id, user):
     return jsonify(briefs=brief.serialize(with_users=True)), 200
 
 
+@main.route('/briefs/<int:brief_id>/suppliers/<int:supplier_code>', methods=['DELETE'])
+def update_brief_remove_rfx_seller(brief_id, supplier_code):
+    updater_json = validate_and_return_updater_request()
+    brief = Brief.query.filter(
```

Comment:
please use service classes for interaction with the db

Reply:
This has been updated in my latest commit.